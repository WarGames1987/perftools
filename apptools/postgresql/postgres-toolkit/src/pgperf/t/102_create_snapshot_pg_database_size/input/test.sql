SELECT pgperf.create_snapshot_pg_database_size(0);
SELECT pgperf.create_snapshot_pg_database_size(1);

SELECT count(*) FROM pgperf.snapshot_pg_database_size;

SELECT pgperf.delete_snapshot_pg_database_size(1);
SELECT count(*) FROM pgperf.snapshot_pg_database_size;

SELECT pgperf.delete_snapshot_pg_database_size(0);
SELECT count(*) FROM pgperf.snapshot_pg_database_size;

