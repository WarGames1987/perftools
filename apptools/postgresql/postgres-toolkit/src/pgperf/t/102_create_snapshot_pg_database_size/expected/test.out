SELECT pgperf.create_snapshot_pg_database_size(0);
 create_snapshot_pg_database_size 
----------------------------------
 t
(1 row)

SELECT pgperf.create_snapshot_pg_database_size(1);
 create_snapshot_pg_database_size 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_database_size;
 count 
-------
     2
(1 row)

SELECT pgperf.delete_snapshot_pg_database_size(1);
 delete_snapshot_pg_database_size 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_database_size;
 count 
-------
     1
(1 row)

SELECT pgperf.delete_snapshot_pg_database_size(0);
 delete_snapshot_pg_database_size 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_database_size;
 count 
-------
     0
(1 row)

